src/jit/importer.cpp

```diff
@@ -1740,47 +1736,55 @@ GenTreePtr          Compiler::impRuntimeLookupToTree(CORINFO_RUNTIME_LOOKUP_KIND
         ctxTree = gtNewLclvNode(info.compTypeCtxtArg, TYP_I_IMPL); // Exact method descriptor as passed in as last arg
     }
 
+#ifdef FEATURE_READYTORUN_COMPILER
+    if (opts.IsReadyToRun())
+    {
+        return impReadyToRunHelperToTree(pResolvedToken, CORINFO_HELP_READYTORUN_GENERIC_HANDLE, TYP_I_IMPL, gtNewArgList(ctxTree), &pLookup->lookupKind);
+    }
```

Comment:
Jit CC guidance - no more than 120 characters per line. I think there are more of these below. Please address them too.